### DISCUSSION TITLE
Management third-party projects

### DISCUSSION THREAD

**Discussion by bitboom** [islet-project/islet#262] [General]
We manage third-party projects (ACS, TF-A, Linux, ...) on the [assets's branch](https://github.com/islet-project/assets).
One of reasons is that it was hard to fork the repository.

But now, we have own organization which means we can manage them as forked git.

So my suggestion is below:
- If third-parties need to be applied our patches, fork them to our organization.
- If third-parties doesn't need any patches, use them directly as git submodule.

I think we could remove almost the 3rd-* branches on [assets](https://github.com/islet-project/assets).
If you have any opinions, feel free to comment :)

**Comment by nook1208** [comment]
> If third-parties need to be applied our patches, fork them to our organization.

I agree with this. Because I remember that there are some cases that we should modify them.
In addition, it seems that forked repositories are better than using branches to manage third-parties.

**Comment by TomaszSwierczek** [comment]
Fully agree with this approach to have an organization with separate repos *where it makes sense*. Especially for elements that are part of Islet stack, do require our custom changes and are not "just" elements of examples (ie. veraison that does need some patch(es) right now but its not Islet dependency). In the future we may want to move Certifier Framework into separate git repo as there's possibility we'll want to extend it during our work on app provisioning. The move towards an organization with set of repos is a very welcome change (like we initially suggested, pity it was just during work on "example" which was remote attestation w/veraison: https://github.com/islet-project/islet/issues/232#issuecomment-1803204675)
